web/src/main/webapp/resources/js/atsy-candidate.js

```diff
@@ -82,12 +82,5 @@ function actionsFormatter(value, row, index) {
              '</a>',
             ].join('');
 }
-
```

Comment:
Why did you refactor this code part from atsy-candidate.js to atsy-settings.js? As I see atsy-candidate.js is used on candidates.jsp.